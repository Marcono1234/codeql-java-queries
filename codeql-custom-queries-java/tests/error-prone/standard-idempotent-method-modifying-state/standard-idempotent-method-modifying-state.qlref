./error-prone/standard-idempotent-method-modifying-state.ql
