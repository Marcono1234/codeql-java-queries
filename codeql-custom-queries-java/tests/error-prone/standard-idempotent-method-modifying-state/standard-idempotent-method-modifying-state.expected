| Test.java:10:20:10:20 | i | Changes state within standard method |
| Test.java:24:20:24:20 | b | Changes state within standard method |
| Test.java:29:20:29:20 | s | Changes state within standard method |
| Test.java:34:20:34:27 | ...++ | Changes state within standard method |
| Test.java:34:32:34:41 | ...=... | Changes state within standard method |
